addons/web/static/src/views/form/form_controller.scss

```diff
@@ -1119,6 +1120,13 @@
 .o_form_view.o_xxs_form_view {
     .o_inner_group {
         @include form-break-table;
+        .o_wrap_field {
+            .o_cell {
+                .o_field_widget {
+                    margin-bottom: 0;
```

Comment:
Thanks for you work @morm-odoo 👍 

Unfortunately this rule will be triggered for address fields when in edit-mode as well, while it should **not**.
Could you address it? Thanks! 

| . | Edit Mode | Read-only |
|--------|--------|--------|
| master | ![image](https://github.com/odoo/odoo/assets/8558351/df4ca46f-eb6c-4ebd-9308-422481154ea2) | ![image](https://github.com/odoo/odoo/assets/8558351/07061886-232c-4e3d-93a5-cccd448b83e2) |
| this PR | ![image](https://github.com/odoo/odoo/assets/8558351/2ba651bd-5f85-4ad1-b20b-a11554fa979d) | ![image](https://github.com/odoo/odoo/assets/8558351/9785c01a-a05e-43d9-85e9-379e36607afb) |

Reply:
@stefanorigano Thanks for the feedback! 
I fixed this issue with adding `&:not(.o_address_format)`, this should avoid the problem in all the address fields.